### PR TITLE
Set API URL in build through environment variables

### PR DESCRIPTION
**Pull request by matthew-a-carr**
## Context

- Sets the API URL for the Spring Boot API via environment variables

## Changes in this pull request

- Updates to pipeline configuration and sets the API_URL environment variable through GitHub Action secrets for each env

## Link to Trello card

- https://trello.com/c/rTWbKmqC/528-hook-up-the-back-office-tool-to-the-api

## Things to check

- [ ] Environment variables have been updated

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const applicationConfig = {
+  apiUrl: process.env.REACT_APP_API_URL,
+};
```
Comment (Ninamma) [inline src/config.ts:3]: 👀 is this like a "nice"/"neat" way of doing env vars, and should we do the same with `AAD_CLIENT_ID`?

Comment (matthew-a-carr) [reply src/config.ts:3]: Oh, I didn't see your auth stuff actually!  It probably makes sense to have everything in one place from an app config perspective so it's easier to find?

I can put the client id in this file and then reference it from the `AuthWrapper`?

**Review by Ninamma — APPROVED** [review]
👍👍

File: src/config.ts (hunk 1 above)
Comment (Ninamma) [reply src/config.ts:3]: Yeah, that sounds great! :D

Comment (matthew-a-carr) [reply src/config.ts:4]: Updated to use the application config 😄

**Pull request merged**